### DISCUSSION TITLE
Using skia in DLL project

### DISCUSSION THREAD

**Discussion by darbdenral** [skia4delphi/skia4delphi#179] [General]
I created a dll in Delphi 10.4 to try and load a lottie file image viewer in an older program by plugin .  So, using the TSkAnimatedImage in an older Delphi project the Lottie picture loads from a local file by json ..  I can see it fine but it won't animate at all.  It works fine inside Delphi 10.4 .  Any idea what I could try?

**Comment by viniciusfbb** [comment]
Can you confirm if the problem has been fixed in newer versions? Remembering that from v4.1.0 the "Enable Skia" was added to the project dlls menu. And remembering that your dll must be in the same folder as sk4d.dll.
